region/maridia/outer/Crab Tunnel.json

```diff
@@ -171,7 +171,14 @@
         {"resetRoom": {
           "nodes": [1]
         }},
-        {"refill": ["PowerBomb"]}
+        {"partialRefill": {
+          "type": "Energy",
+          "limit": 200
```

Comment:
```suggestion
          "limit": 100
```
There's only one crab per reset. This feels like more room resets than any of the other strats. You could add a higher refill if there was a super used.